ros/src/tl_detector/light_classification/tl_classifier.py

```diff
@@ -19,18 +29,35 @@ def get_classification(self, image):
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-        state = TrafficLight.UNKNOWN
+        # state = TrafficLight.UNKNOWN
 
-        red_area, red_image = self.get_colored_area(image, self.lower, self.upper)
+        # red_area, red_image = self.get_colored_area(image, self.lower, self.upper)
 
         # Needs careful tuning for the number of red pixesls
-        red_pixels = 40
+        # red_pixels = 40
+
+        # if red_area > red_pixels:
+        #     state = TrafficLight.RED
+
+        # return state
+
 
-        if red_area > red_pixels:
-            state = TrafficLight.RED
+        image = self.process_image(image)
```

Comment:
Not really a good practice to reuse same variable name given content changed. It's not the end of the world and if you insist, you can keep it that way, but it would be better to change this to
```processed_image = self.process_image(image)```